database-postgresql/helpers.js

```diff
@@ -0,0 +1,62 @@
+
+const client = require('./connection');
+
+const getPhotos = (trailId, sortOrder, callback) => {
+  let sortOrderStatement = sortOrder ? `ORDER BY upload_date ${sortOrder}` : '';
+
+  let getPhotosPSQLStatement = `SELECT * FROM trailphotos WHERE trail_id = $1` + sortOrderStatement;
+
+  client.query(getPhotosPSQLStatement, [trailId], (err, res) => {
+    if(err) throw err;
+    let result = {
+      data: res.rows.map(item => {
```

Comment:
A named function here would be helpful for testing, readability, and possibly re-usability.